### PR TITLE
Enable some CPU tests on draft mode

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

See title. The list and parameterizations of jobs is be unchanged.

### CODE REVIEW

File: .github/workflows/ci-pytorch-test-full.yml [hunk 1]
```diff
@@ -36,17 +36,17 @@ jobs:
           - {os: "ubuntu-20.04", python-version: "3.8", pytorch-version: "1.10"}  # ex-conda
           - {os: "ubuntu-20.04", python-version: "3.9", pytorch-version: "1.11"}  # ex-conda
           - {os: "ubuntu-20.04", python-version: "3.9", pytorch-version: "1.12"}  # ex-conda
-          - {os: "ubuntu-20.04", python-version: "3.10", pytorch-version: "1.12"}
+          - {os: "ubuntu-20.04", python-version: "3.10", pytorch-version: "1.12", draft: "true"}
           - {os: "macos-11", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
```
Comment (carmocca) [inline .github/workflows/ci-pytorch-test-full.yml:40]: @akihironitta unrelated question. why do we not set `requires` for all jobs?

**Comment by carmocca** [comment]
Looks like this is not supported at the moment: https://github.com/actions/runner/issues/1985

**Review by Borda — COMMENTED** [review]
you would need to make the workflow abstract and call it in two jobs, one running with any other running without draft

File: .github/workflows/ci-pytorch-test-full.yml (hunk 1 above)
Comment (akihironitta) [reply .github/workflows/ci-pytorch-test-full.yml:40]: @carmocca I'm sorry I missed your message here. `requires: "latest"` wasn't used in any steps' conditions in the workflow, so I only kept `requires: "oldest"`. We can add those back here (or remove them in other workflows, too) for consistency.

Comment (carmocca) [reply .github/workflows/ci-pytorch-test-full.yml:40]: It's okay. Now I see that it's simpler to omit it

**Pull request closed without merging**